src/views/annual-report/2020/annual-report.scss

```diff
@@ -0,0 +1,2327 @@
+
+                    .comment-text {
+                        margin-left: 12px;
+                    }
+                }
+            }
+        }
+
+        @media #{$small} {
+            .inner {
+                overflow: hidden;
+            }
+
+            .hero {
+                width: 540px;
+            }
+
+            .community-moderation, .community-guidelines, .community-engagement {
+                width: 300px;
+                text-align: left;
+            }
+
+            ul {
+                margin: 0;
+                padding-left: 24px;
+            }
+
+            li {
+                margin-bottom: 0;
+            }
+        }
+    }
+
+    .video-container {
+        width: 100%;
+        display: flex;
+        justify-content: center;
+        margin-top: 30px;
+    }
+
+    .video-thumbnail .button{
+        top: -113px;
+        font-size: 15px;
+        position: relative;
+        border-radius: 3px;
+        @media #{$medium}{
+            top: -99px;
+        }
+        @media #{$small}{
+            top: -107px;
+            top: -74px;
+            font-size: 12px;
+        }
+    }
+
+    .themes .video-thumbnail .button{
+        top: -119px;
+        @media #{$medium}{
+            top: -107px;
+        }
+        @media #{$small}{
+            top: -105px;
+            top: -74px;
+            font-size: 12px;
+        }
+    }
+
+    .tweet-container{
+        background-image: url("/images/annual-report/2020/adaptation/tweet-bg.svg");
+        background-size: contain;
+        background-position: center;
+        background-repeat:no-repeat;
+        width: 100%;
+        max-width: 900px;
+        margin: -50px auto 0;
+        .tweets{
+            max-width: 600px;
+            width: 100%;
+            display: flex;
+            margin: auto;
+            justify-content: space-around;
+            align-items:flex-start;
+            @media #{$small} {
+                flex-direction: column;
+            }
+            .twitter-tweet{
+                width: 100%;
+            }
+        }
+    }
+
+    .video-background {
+        width: 100%;
+        height: 500px;
+        overflow: hidden;
+
+        background-size: auto 100%;
+        background-repeat: no-repeat;
+        background-position: 50% 50%;
+
+        display: flex;
+        align-items: center;
+        justify-content: center;
+
+        &.blm {
+            background-image: url("/images/annual-report/initiatives/BLM Video Splash.svg");
+        }
+
+        &.abhi {
+            background-image: url("/images/annual-report/initiatives/Abhi Video Splash.svg");
+
+            .button {
+                background-color: $ui-aqua;
+            }
+        }
+
+        &.cps {
+            background-image: url("/images/annual-report/initiatives/schools/CPS Story/CPS Video Splash.svg");
+
+            .button {
+                background-color: $ui-purple-dark;
+            }
+        }
```

Comment:
These look like they're the ones for the 2019 annual report (?)

Reply:
yep, this was a mistake. This has been removed.